clang/test/CodeGenSYCL/disable_loop_pipelining.cpp

```diff
@@ -29,8 +29,8 @@ int main() {
   return 0;
 }
 
-// CHECK: define dso_local spir_kernel void @{{.*}}test_kernel1() #0 {{.*}} !disable_loop_pipelining ![[NUM5:[0-9]+]]
+// CHECK: define dso_local spir_kernel void @{{.*}}test_kernel1() #0 {{.*}} !disable_kernel_pipelining ![[NUM5:[0-9]+]]
```

Comment:
Is it possible to rename this to "!pipeline_kernel", and to flip the argument? We believe it may convey information better about what to do with the kernel when the property is absent.

Reply:
You may also want to rename this file to match with the new name.